pom.xml

```diff
@@ -32,11 +32,11 @@
         <mapstruct.version>1.6.3</mapstruct.version>
         <opentelemetry-instrumentation-bom.version>2.16.0</opentelemetry-instrumentation-bom.version>
         <!--- CH -->
-        <structured-logging.version>3.0.33</structured-logging.version>
-        <private-api-sdk-java.version>4.0.316</private-api-sdk-java.version>
-        <api-sdk-manager-java-library.version>3.0.7</api-sdk-manager-java-library.version>
-        <api-security-java.version>2.0.8</api-security-java.version>
-        <api-helper-java.version>3.0.1</api-helper-java.version>
+        <structured-logging.version>3.0.38</structured-logging.version>
+        <private-api-sdk-java.version>4.0.333</private-api-sdk-java.version>
```

Comment:
there is a newer version of private-api-sdk-java that has security issues removed 4.0.337

Reply:
thanks have changed to 4.0.337 as advised